app/assets/javascripts/discourse/app/widgets/hooks.js

```diff
@@ -262,6 +264,12 @@ WidgetClickHook.setupDocumentCallback = function () {
     });
   });
 
+  $(document).on("touchmove.discourse-widget", (e) => {
```

Comment:
Introducing a global `touchmove` listener will have a pretty bad performance impact because it blocks scroll on mobile devices. Instead, we should use the same technique that we have for touchstart:

https://github.com/discourse/discourse/blob/9e8010df8bfd66dcbb7339b8bf18b01b26c7350f/app/assets/javascripts/discourse/app/widgets/hooks.js#L68-L75

(only adds the listener when needed, and scopes it to the specific element)

Reply:
Yes I was reading at this while answering on the other PR, I don't think we had this last time I worked on this.

Reply:
Used same pattern in: https://github.com/discourse/discourse/pull/22635/commits/475f14553e843744833382be4bedb02a71a1acf1

I checked and it doesn't seem to change the behavior I explained in the other  PR

Reply:
I guess that given this custom code for touch start and now touch move doesn't use `nodeCallback()` it won't ever re-render for these events itselves.